realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -585,6 +622,7 @@ private void emitCopyOrUpdateMethod(JavaWriter writer) throws IOException {
                 .beginControlFlow("if (rowIndex != TableOrView.NO_MATCH)")
                     .emitStatement("realmObject = new %s()", Utils.getProxyClassName(className))
                     .emitStatement("realmObject.realm = realm")
+                    .emitStatement("realmObject.updateColumnInfo(realm.getColumnInfo(%s.class))", className)
```

Comment:
Oh, I already implemented the way of constructor.
If it's better to use reimplement with `RealmObjectProxy`,please comment.